### PR TITLE
Add Chrome/Safari versions for HashChangeEvent API

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR adds real values for Chrome and Safari for the `HashChangeEvent` API, based upon manual testing.

Test Code Used:
```js
if (location.hash === '#foobar') {
	location.hash = '#foo';
} else {
	location.hash = '#foobar';
}

window.addEventListener('hashchange', function(event) {
	alert(event.oldURL);
	alert(event.newURL);
});
```

### CODE REVIEW

File: api/HashChangeEvent.json [hunk 1]
```diff
@@ -121,22 +121,22 @@
               "version_added": false
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤15"
```
Comment (foolip) [inline api/HashChangeEvent.json:124]: Can you test Presto while you have the test handy?

**Review by foolip — CHANGES_REQUESTED** [review]

File: api/HashChangeEvent.json [hunk 2]
```diff
@@ -121,22 +121,22 @@
               "version_added": false
             },
             "opera": {
-              "version_added": true
+              "version_added": "15"
```
Comment (foolip) [inline api/HashChangeEvent.json:124]: Support for the event interface but none of the properties seemed suspect so I tested Opera 12.16 a bit. `document.createEvent('HashChangeEvent').newURL` is the empty string meaning this was supported in Presto.

**Review by foolip — CHANGES_REQUESTED** [review]

File: api/HashChangeEvent.json (hunk 2 above)
Comment (foolip) [reply api/HashChangeEvent.json:124]: Oh, and actually https://staging-dot-mdn-bcd-collector.appspot.com/tests/api/HashChangeEvent can test this now with your recent custom test additions.

File: api/HashChangeEvent.json [hunk 3]
```diff
@@ -121,22 +121,22 @@
...
-              "version_added": true
+              "version_added": "≤12.1"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": true
+              "version_added": "5.1"
```
Comment (foolip) [inline api/HashChangeEvent.json:130]: This also doesn't quite match `onhashchange` data. Can you make bring them into alignment?

File: api/HashChangeEvent.json [hunk 4]
```diff
@@ -103,10 +103,10 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/browsing-the-web.html#dom-hashchangeevent-newurl-dev",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "8"
```
[the remark this answers is not in the document]
Comment (vinyldarkscratch) [reply api/HashChangeEvent.json:106]: It was just a plain `Event` beforehand; same with Safari.

**Review by foolip — APPROVED** [review]

**Pull request merged**